main/src/ca/uwaterloo/flix/api/lsp/provider/CompleteProvider.scala

```diff
@@ -21,10 +21,39 @@ object CompleteProvider {
     // NB: Please keep the list alphabetically sorted.
 
     // Keywords:
-    CompletionItem("namespace", "namespace", None, None, CompletionItemKind.Keyword, InsertTextFormat.PlainText, List("{", "}")),
-
-    // Keyword-like (e.g. println):
-    CompletionItem("println", "println", None, None, CompletionItemKind.Keyword, InsertTextFormat.PlainText, List("(", ")")),
+    CompletionItem("as", "as", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("BigInt", "BigInt", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("Bool", "Bool", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("case", "case", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.Snippet, Nil),
+    CompletionItem("chan", "chan", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("Char", "Char", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("deref", "deref", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("false", "false", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, List(";")),
+    CompletionItem("Float", "Float", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
```

Comment:
Remove `Float` (we are planning to remove it from the language, forcing the programmer to explicitly write Float32 or Float64). But of course there is no way for you to know that :) The same is true for `Int`.